schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-dataset.xsl

```diff
@@ -0,0 +1,260 @@
+                as="xs:string"
+                select="'http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/uom/ML_gmxUom.xml#'"/>
+  <!--
+  RDF Property:	dcat:spatialResolutionInMeters
+  Definition:	Minimum spatial separation resolvable in a dataset, measured in meters.
+  Range:	rdfs:Literal typed as xsd:decimal
+  Usage note:	If the dataset is an image or grid this should correspond to the spacing of items.
+  For other kinds of spatial datasets, this property will usually indicate the smallest distance between items in the dataset.
+  -->
+  <xsl:template mode="iso19115-3-to-dcat"
+                match="mri:spatialResolution/*/mri:distance">
+    <xsl:variable name="unitId"
+                  as="xs:string?"
+                  select="if (starts-with(*/@uom, $isoUnitCodelistBaseUri))
+                            then substring-after(*/@uom, $isoUnitCodelistBaseUri)
+                            else */@uom"/>
+
+    <xsl:variable name="knownCode"
+                  as="xs:string?"
+                  select="$unitCodes[text() = $unitId]/@id"/>
+
+    <xsl:variable name="unit" as="xs:string?"
+                  select="if ($knownCode) then $knownCode else */@uom"/>
+
+    <xsl:variable name="conversionFactor"
+                  as="xs:decimal?"
+                  select="$unitConversionFactorForMeter[@id = $unit]/text()"/>
+
+    <xsl:choose>
+      <xsl:when test="($conversionFactor)
+                      and */text() castable as xs:decimal">
+        <dcat:spatialResolutionInMeters rdf:datatype="http://www.w3.org/2001/XMLSchema#decimal">
+          <xsl:value-of select="xs:decimal(.) * $conversionFactor"/>
+        </dcat:spatialResolutionInMeters>
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:comment>WARNING: Spatial resolution only supported in meters. <xsl:value-of select="concat(*/text(), ' ', */@uom)"/> is ignored (can be related to unknown unit or no conversion factor or not a decimal value).</xsl:comment>
+      </xsl:otherwise>
+    </xsl:choose>
+
+    <!-- TODO: GeoDCAT-AP
+    <dqv:hasQualityMeasurement>
+      <dqv:QualityMeasurement>
+        <dqv:isMeasurementOf>
+          <dqv:Metric rdf:about="{$geodcatap}spatialResolutionAsDistance"/>
+        </dqv:isMeasurementOf>
+        <dqv:value rdf:datatype="{$xsd}decimal"><xsl:value-of select="."/></dqv:value>
+        <sdmx-attribute:unitMeasure rdf:resource="{$uom-km}"/>
+      </dqv:QualityMeasurement>
+    </dqv:hasQualityMeasurement>
+    -->
+
+  </xsl:template>
+
+
+  <!--
+  RDF Property:	dcterms:accrualPeriodicity
+  Definition:	The frequency at which a dataset is published.
+  Range:	dcterms:Frequency (A rate at which something recurs)
+  Usage note:	The value of dcterms:accrualPeriodicity gives the rate at which the dataset-as-a-whole is updated. This may be complemented by dcat:temporalResolution to give the time between collected data points in a time series.
+  -->
+  <xsl:variable name="isoFrequencyToDublinCore"
```

Comment:
I am under the impression that this mapping is not applied. On my experiments, `annually` is kept as-is instead of being replaced by `ANNUAL` in the dcat output (default DCAT `outputSchema=http://www.w3.org/ns/dcat#`)

Reply:
![image](https://github.com/geonetwork/core-geonetwork/assets/4491521/c90e07ed-a51a-497a-9c71-5f2144e72633)

Reply:
Previous remark applies to csw-dcat output. 

But indeed it works when formattin the file through the dcat download formatter `/formatters/dcat` (download button)

Reply:
`http://www.w3.org/ns/dcat#` correspond to current conversion (see dcat-brief.xsl)

Use `http://www.w3.org/ns/dcat#core` when using CSW
 eg.
http://localhost:8080/geonetwork/srv/fre/csw?SERVICE=CSW&VERSION=2.0.2&REQUEST=GetRecordById&ID=da165110-88fd-11da-a88f-000d939bc5d8&outputschema=http%3A%2F%2Fwww.w3.org%2Fns%2Fdcat%23core

Maybe the current conversion should be renamed to `http://www.w3.org/ns/dcat#legacy` ? but that will alter current apps using the current DCAT mapping